CHANGELOG.md

```diff
@@ -9,8 +9,45 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ### Added
 
 ### Fixed
-- fix: queries with up to 65535 (inclusive) parameters are supported now (previous limit was 32767) [PR #2525](https://github.com/pgjdbc/pgjdbc/pull/2525)
-- fix: workaround JarIndex parsing issue by using groupId/artifactId-version directory namings. Regression since 42.2.13. [PR #2531](https://github.com/pgjdbc/pgjdbc/pull/2531), [issue #2527](https://github.com/pgjdbc/pgjdbc/issues/2527)
+
+[42.4.0] (2022-06-08 15:39:02 -0400)
+### Changed
+- fix: added GROUP_STARTUP_PARAMETERS boolean property to determine whether or not to group 
+startup parameters in a transaction or not fixes [Issue #2425](https://github.com/pgjdbc/pgjdbc/issues/2497)  
+pgbouncer cannot deal with transactions in statement pooling mode [PR #2425](https://github.com/pgjdbc/pgjdbc/pull/2425)
+
+### Added
+
+### Fixed
+- chore: Make the readme version agnostic [PR #2540](https://github.com/pgjdbc/pgjdbc/pull/2540)
+- fix: queries with up to 65535 (inclusive) parameters are supported now (previous limit was 32767) 
+[PR #2525](https://github.com/pgjdbc/pgjdbc/pull/2525), [Issue #1311](https://github.com/pgjdbc/pgjdbc/issues/1311)
+- fix: workaround JarIndex parsing issue by using groupId/artifactId-version directory namings. 
+Regression since 42.2.13. [PR #2531](https://github.com/pgjdbc/pgjdbc/pull/2531), [issue #2527](https://github.com/pgjdbc/pgjdbc/issues/2527)
+- chore: skip running publishToNexus on forks if NEXUS_* secrets are not set
+- test: remove SlowTests from tests that are fast
```

Comment:
This might go with the rest test modifications under a single "improve test code robustness in various locales, JDKs"